x-pack/solutions/search/test/serverless/functional/test_suites/navigation.ts

```diff
@@ -230,34 +231,72 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
       await solutionNavigation.sidenav.expectLinkExists({ text: 'Performance' });
       await solutionNavigation.sidenav.expectLinkExists({ text: 'Billing and subscription' });
 
+      if (isV1) {
+        // v2 ignores sections
+        await solutionNavigation.sidenav.expectLinkExists({ text: 'Relevance' });
+        await solutionNavigation.sidenav.expectLinkExists({ text: 'Build' });
```

Comment:
We could move the home check here as well vs changing to check for a deepLinkId since we already have tests that verify the expected deepLinkIds. This tests is very specifically checking for sidenav links with expected copy.

```suggestion
        await solutionNavigation.sidenav.expectLinkExists({ text: 'Home' });
        // v2 ignores sections
        await solutionNavigation.sidenav.expectLinkExists({ text: 'Relevance' });
        await solutionNavigation.sidenav.expectLinkExists({ text: 'Build' });
```